rdbi-parent/pom.xml

```diff
@@ -91,12 +91,12 @@
             <dependency>
                 <groupId>com.google.code.findbugs</groupId>
                 <artifactId>jsr305</artifactId>
-                <version>2.0.1</version>
+                <version>3.0.1</version>
             </dependency>
             <dependency>
                 <groupId>joda-time</groupId>
                 <artifactId>joda-time</artifactId>
-                <version>2.9.4</version>
+                <version>2.9.9</version>
```

Comment:
what would it take to just drop this dependency? i've been slowly removing it in favor of java8 date utils.

Reply:
lololol now you're talkin'!